model-optimizer/extensions/back/compress_quantized_weights.py

```diff
@@ -218,3 +218,56 @@ def replace_pattern(self, graph: Graph, match: Dict[str, Node]):
 
         self.quantize_data(fake_quantize, dst_type, quantized_type, mode)
         self.dequantize_data(fake_quantize, dst_type, quantized_type)
+
+
+class ZeroPointOptimizer(BackReplacementPattern):
+    enabled = True
+    force_clean_up = True
+
+    def run_after(self):
+        return [CompressQuantizeWeights]
+
+    def pattern(self):
```

Comment:
when is this transformation triggered? whenever does this pattern match?

Reply:
triggered for compressed weights as it is written in the pattern

Reply:
nit: I see that it is applied to `weights.dtype == np.int8`. I think it makes sense to make a note in the description that it is applied only to weights of `np.int8`.